Apps/W1/APIV2/test/src/APIV2FALocationsE2E.Codeunit.al

```diff
@@ -0,0 +1,161 @@
+codeunit 139900 "APIV2 - FA Locations E2E"
```

Comment:
These cases look rather like unit tests, not E2E. The name could be misleading.

Reply:
In general I agree but - this is exact copy of the test https://github.com/microsoft/ALAppExtensions/blob/main/Apps/W1/APIV2/test/src/APIV2LocationsE2E.Codeunit.al 

And all tests in the module are called E2E - means:
1. Either we keep it to have the same names for all
2. MS should change other tests too